src/app/api/alerts/routing.py

```diff
@@ -0,0 +1,50 @@
+from typing import List
+from fastapi import APIRouter, HTTPException, Path
+from app.api import crud
+from app.db import alerts
+from .schemas import AlertIn, AlertOut
+
+
+router = APIRouter()
+
+
+@router.post("/", response_model=AlertOut, status_code=201)
+async def create_alert(payload: AlertIn):
+    _id = await crud.post(payload, alerts)
+
+    return {**payload.dict(), "id": _id}
+
+
+@router.get("/{id}/", response_model=AlertOut)
+async def get_alert(id: int = Path(..., gt=0),):
```

Comment:
I just discovered the "ellipsis" feature thanks to your commit.
What does it do here?

Reply:
It specifies that the argument is required